R/geom-histogram.r

```diff
@@ -2,28 +2,28 @@
 #'
 #' Visualise the distribution of a single continuous variable by dividing
 #' the x axis into bins and counting the number of observations in each bin.
-#' Histograms (`geom_histogram`) display the count with bars; frequency
-#' polygons (`geom_freqpoly`) display the counts with lines. Frequency
+#' Histograms (`geom_histogram()`) display the count with bars; frequency
```

Comment:
You have "Histograms…display the count" and "frequency polygons…display the count*s*"— I'm not partial to one or the other, I just think they should be the same.